app.py

```diff
@@ -1,10 +1,11 @@
 from bottle import request, route, run, template, post, TEMPLATE_PATH, abort, redirect
 from marshmallow import Schema, fields
-import dbfunctions
+from dbfunctions import Wikidb
 
 TEMPLATE_PATH.append('templates')
 DB_PATH = './wiki.db'
 
+db = Wikidb(db_path=DB_PATH)
```

Comment:
This just makes the most sense to me to do first. It's the logical first step that must be completed for our feature set and the most likely point to fail, I figure failing sooner is better than later. I also gravitate to it because it's what I've always what I've done. Not a good reason, but a reason.
I am interested in the specific failure cases that require not doing this, but having no concrete reason one way or the other, I'll just move it back based on your significantly greater experience.

I actually expected the linter to yell at me since moving this to only occur when __main__ means db doesn't exist when importing and all the functions are broken unless supplemented somewhere else. Linters not that smart however (which is always the first thing I notice about linters...)